integration_test/third_party_apps_data/applications/postgresql/metadata.yaml

```diff
@@ -31,31 +31,145 @@ minimum_supported_agent_version:
 supported_operating_systems: linux
 supported_app_version: ["10.18+"]
 expected_metrics:
-  - type: workload.googleapis.com/postgresql.backends
+  - kind: GAUGE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.backends
+    value_type: INT64
+    representative: true
+  - kind: CUMULATIVE
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.allocated
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      source: backend|backend_fsync|checkpoints|bgwriter
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.writes
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      type: requested|scheduled
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.checkpoint.count
```

Comment:
The scope [doc](https://docs.google.com/document/d/1M9r8CqZl2W9fYz8brzlWntEAqJUC1K43LHGvGkbpiPs/edit) has this as `workload.googleapis.com/postgresql.bgwriter.checkpoints`

Is this an intentional change to the scope doc? If so, can we send a suggested edit to the scope doc and have the approver double check it? If not, can we fix the translation so it matches the scope doc?

Reply:
It appears that this metric name matches when I looked at the doc? Do you mind taking a second pass to make sure its not `postgresql.bgwriter.checkpoint.count`?

Reply:
It looks to me like it says the otel metric name is `postgresql.bgwriter.checkpoint.count` and that we're expecting to emit `postgresql.bgwriter.checkpoints` from the Ops Agent.

Reply:
Ahh I see I was looking at the otel vs ops agent metric now, thanks for the catch! Will make a suggestion!